src/Psalm/Internal/Analyzer/Statements/Expression/Call/Method/MethodCallReturnTypeFetcher.php

```diff
@@ -97,16 +98,14 @@ public static function fetch(
 
         if ($premixin_method_id->method_name === 'getcode'
             && $premixin_method_id->fq_class_name !== Exception::class
+            && $premixin_method_id->fq_class_name !== RuntimeException::class
+            && $premixin_method_id->fq_class_name !== PDOException::class
```

Comment:
Please use "PDOException" as a string. We don't want to introduce a false dependency when it's not needed

Reply:
It was already used with `$premixin_method_id->fq_class_name === PDOException::class`, you said it was ok for FQCN: https://github.com/vimeo/psalm/pull/7390#discussion_r784799089

Reply:
Yeah, but I believe we went back on another PR, let me check that

Reply:
nvm, this was for constants for allowing to use them without having the extension: https://github.com/vimeo/psalm/pull/7107/files#r765332473

It shouldn't be an issue for ::class, forget I said anything :)

Reply:
It was changed to a string in #7648.